aws_gate/cli.py

```diff
@@ -29,14 +30,18 @@ def main():
     parser.add_argument('-v', '--verbose', help='increase output verbosity',
                         action='store_true')
     parser.add_argument('--version', action='version', version='%(prog)s {version}'.format(version=__version__))
-    subparsers = parser.add_subparsers(title='subcommands', dest='subcommand', metavar='{session}')
+    subparsers = parser.add_subparsers(title='subcommands', dest='subcommand', metavar='{session, ls}')
 
     # 'session' subcommand
     session_parser = subparsers.add_parser('session', help='Open new session on instance and connect to it')
     session_parser.add_argument('-p', '--profile', help='AWS profile to use', default=default_profile)
     session_parser.add_argument('-r', '--region', help='AWS region to use', default=default_region)
     session_parser.add_argument('instance_name', help='Instance we wish to open session to')
 
+    ls_parser = subparsers.add_parser('ls', help='List available instances and connect to it')
```

Comment:
From what I understood from the code, list only lists available instances, it doesn't connect to them.